pkg/plugin/plugin.go

```diff
@@ -359,8 +361,44 @@ func CmdAdd(args *skel.CmdArgs) error {
 			ipc.Interface = current.Int(0)
 		}
 
+		// wait until OF port link state becomes up. This is needed to make
+		// gratuitous arp for args.IfName to be sent over ovs bridge
+		for i := 0; i < 5; i++ {
```

Comment:
Could you please define a helper function for this `waitLinkUp` taking the number of retries and interval? To keep the main flow of this (already too big) function clean.

Reply:
Good thought! done.